great_expectations/expectations/metrics/metric_provider.py

```diff
@@ -1,12 +1,14 @@
+import copy
 import logging
 from functools import wraps
 from typing import Callable, Optional, Type, Union
 
 from great_expectations.core import ExpectationConfiguration
-from great_expectations.core.domain_types import MetricDomainTypes
+from great_expectations.core.util import nested_update
```

Comment:
This is coming from develop but is also not used there

Reply:
It is all from develop...